packages/tooltip/README.md

```diff
@@ -19,3 +19,13 @@ Tooltips provide additional information or clarification about a UI element when
 
 - Use hint text below the field to provide information about what input is expected.
 - Use a Tooltip to provide additional explanation about the field. Usually, Tooltips are triggered in form fields by an icon in the field's label.
+
+### Using Icons with Tooltips
```

Comment:
❓ Should we make it possible to use any icon? Just because it's still an open source library and other people might would want to use these differently.
Additionally, if we ever decide to use other icons we do not need to change the code here.

Reply:
I did consider this but tooltip icons are really only ever these two options. This is where it helps with the consistency part. Sometimes it's good to limit options to help consumers make the right decisions. If you wanted a completely different icon you could just pass in the `Icon` component into the trigger and i'm not sure how that would make this component much different than using that alone.